components/chips/mdc-chip.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="mdc-chip" :class="classes" :style="styles" tabindex="0" @click="dispatchEvent">
+    <i class="material-icons mdc-chip__icon mdc-chip__icon--leading" v-if="icon && !trailing">{{icon}}</i>
```

Comment:
here we need to find a way to use awesome fonts as well.

Reply:
I think this can be done by added a new attribute `iconClasses` or similar and a `computed` for some logic.

Will look into this.